tox.ini

```diff
@@ -71,12 +71,12 @@ commands =
 [testenv:flake8]
 recreate = True
 commands =
-    pip install --upgrade pip six setuptools
+    pip install -r requirements/system.txt
     pip install -r requirements/flake8.txt
     make flake8
 
 [testenv:docs]
 commands =
-    pip install --upgrade pip six setuptools
+    pip install -r requirements/system.txt
```

Comment:
With that, we should also be able to clean up deps in `docs.txt` right ? The same way you did in `prod.txt`

Reply:
hell yeah, I'm on it.